src/System Application/App/Entity Text/src/EntityTextScenarioExt.EnumExt.al

```diff
@@ -0,0 +1,9 @@
+namespace System.Text;
+
+enumextension 2014 "Entity Text Scenario Ext." extends "Entity Text Scenario"
```

Comment:
Extensions are rather limiting. You can have 1 per app. Keep this in mind. Maybe this was changed with recent modern dev work, but I believe it was not, try to define two enum extensions in the same app and see if it will compile.